scripts/import_from_ena.py

```diff
@@ -27,22 +33,57 @@
 DATA_SOURCES = ['fastq', 'sra', 'cram_index']
 DATA_TYPES = ['ftp', 'galaxy', 'aspera']
 
+logger = logging.getLogger(__name__)
```

Comment:
You can instantiate logger from utils as I do this for other scripts, for example logger = create_logging_instance('create_summary'), it takes name of script as parameter

Reply:
done in b9337f6